### PR TITLE
add genre entry point for music

### PR DESCRIPTION
**Pull request by aronparsons**

### CODE REVIEW

File: 1080i/Includes_Variables.xml [hunk 1]
```diff
@@ -169,6 +169,7 @@
     <value condition="Skin.HasSetting(MusicEntry.Playlists)">ActivateWindow(Music,Playlists,return)</value>
     <value condition="Skin.HasSetting(MusicEntry.Files)">ActivateWindow(Music,Files,return)</value>
     <value condition="Skin.HasSetting(MusicEntry.Albums)">ActivateWindow(Music,Albums,return)</value>
+    <value condition="Skin.HasSetting(MusicEntry.Genres)">ActivateWindow(Music,Genres,return)</value>
     <value>ActivateWindow(Music,Artists,return)</value>
   </variable>
```
Comment (joethepartylion) [inline 1080i/Includes_Variables.xml:174]: You also need to add a value for MusicEntryPointLabel2 to show the selected setting within the settings menu

**Comment by aronparsons** [comment]
@joethepartylion you're saying there's something else other than the other part of this commit where the additional option is appended to the SelectBool function for "music entry points"? https://github.com/jester-xbmc/skin.amber/pull/62/files#diff-b298c5cf01e822a7aedc064d69885d91R986

**Comment by joethepartylion** [comment]
Yes on line 985 label2 gets its setting from the variable MusicEntryPointLabe2, this is what is displayed to the right of the setting to show what it is currently set to. You just need to add a new value with the same condition as the one you've created but with the localised string for Genre as the value.

**Comment by joethepartylion** [comment]
That is line 985 of settings.xml

**Comment by aronparsons** [comment]
I see.  Totally glanced over the second variable declaration.

**Comment by joethepartylion** [comment]
No problem, thanks for fixing the description too :-)

**Comment by jester-xbmc** [comment]
Welcome to the fun of skin editing :) let me know once this is whipped into shape and i'll merge.
Cheers

**Comment by joethepartylion** [comment]
All looks good to me now. I'll add to TVshows and Movies too for completeness in my Krypton3 branch when I send that over, just making sure the shelves are fully working now I've removed the skin widgets dependency.

**Pull request merged**